README.rst

```diff
@@ -49,26 +49,27 @@ PennyLane-Lightning high performance simulators include the following backends:
 
 .. header-end-inclusion-marker-do-not-remove
 
 The following table summarizes the supported platforms and the primary installation mode:
 
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
-|           | L-Qubit | L-GPU  | L-GPU (MPI) | L-Kokkos (OMP) | L-Kokkos (CUDA) | L-Kokkos (HIP) |
-+===========+=========+========+=============+================+=================+================+
-| Linux x86 | pip     | pip    | source      | pip            | source          | source         |
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
-| Linux ARM | pip     | source |             | pip            | source          | source         |
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
-| Linux PPC | pip     | source |             | pip            | source          | source         |
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
-| MacOS x86 | pip     |        |             | pip            |                 |                |
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
-| MacOS ARM | pip     |        |             | pip            |                 |                |
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
-| Windows   | pip     |        |             |                |                 |                |
-+-----------+---------+--------+-------------+----------------+-----------------+----------------+
++-----------+---------+--------+-------------+----------------+-----------------+----------------+----------------+
+|           | L-Qubit | L-GPU  | L-GPU (MPI) | L-Kokkos (OMP) | L-Kokkos (CUDA) | L-Kokkos (HIP) |    L-Tensor    |
++===========+=========+========+=============+================+=================+================+================+
+| Linux x86 | pip     | pip    | source      | pip            | source          | source         |     source     |
++-----------+---------+--------+-------------+----------------+-----------------+----------------+----------------+
+| Linux ARM | pip     | source |             | pip            | source          | source         |                |
```

Comment:
I guess we have not tested it, but could we quickly check whether it works on our GraceHopper machine?
```suggestion
| Linux ARM | pip     | source |             | pip            | source          | source         |  source   |
```

Reply:
There are randomly failures for both LGPU and LTensor when @maliasadi tested them on GH. It's on the roadmap of Q3.